### PR TITLE
Revert "feat: Add an `adm_mobile_max_tiles` setting (DISCO-2306, #529)"

### PR DESCRIPTION
**Pull request by ncloudioj**
This reverts commit 23f5ae6993ed93744bf92bfb3bbca3c900a1eaeb.

We will wait for the dismissal telemetry to hit the road in Fx 113 to enable this change in Contile.

### CODE REVIEW

**Review by pjenvey — COMMENTED** [review]
Alternatively we could set both settings to the original 3 value in the helm chart.

File: src/settings.rs [hunk 1]
```diff
@@ -103,14 +103,12 @@ pub struct Settings {
     pub adm_sub1: Option<String>,
     /// adm Endpoint URL
     pub adm_endpoint_url: String,
-    /// max number of tiles returned to non-mobile clients (default: 3)
-    pub adm_max_tiles: u8,
```
Comment (linabutler) [inline src/settings.rs:107]: I am a little surprised that it's easier to make a code change than to set an environment variable to override this, though. Isn't this setting controlled by `CONTILE_ADM_MAX_TILES`? How come we can't set it to `2` to override the new default of 3?

**Comment by ncloudioj** [comment]
You know what, it's very likely that we will bump from 3 to 4 or other values in the future, we will be using the EnvVar for that anyway. I will file a PR to add that to the cloud-infra repo instead.

Thanks for the reminder.

**Comment by pjenvey** [comment]
@ncloudioj FYI we (the contile-reviewers group) do now have ownership (can approve) files under the `topsites/` directory so they don't even need approval. I think this settings addition hasn't rolled out to stage/prod yet so I think we just need to remind them to incorporate the helm change for the next release (which might even happen automatically once it's merged).

**Comment by ncloudioj** [comment]
kk, close this to favor https://github.com/mozilla-services/cloudops-infra/pull/4853.

**Pull request closed without merging**